openssl-sys/src/lib.rs

```diff
@@ -2252,6 +2252,14 @@ extern "C" {
                                                               arg: *mut c_void)
                                                               -> c_int,
                                             arg: *mut c_void);
+    pub fn SSL_CTX_set_psk_client_callback(ssl: *mut SSL_CTX,
+                                           psk_client_cb: extern "C" fn(*mut SSL,
```

Comment:
I wasn't entirely sure of the convention here. Is `Option<...>` used for nullable function pointers?